docs/pipelines/add-pipeline.md

```diff
@@ -19,8 +19,11 @@ required_permissions:
 1. In the list of applications find the application where you want to add a pipeline and click `DETAILS`
 1. On the upper-right corner of the applications details view click `SETTINGS`
 1. Use the drop-down menu to choose `PIPELINES`
-1. On the right corner of the list click `ADD PIPELINE`
-1. Follow the prompts to complete the configuration of the new `Pipeline`
+1. On the right corner of the list click `MANAGE`
+1. Provide a name for the new pipeline
+1. Select the `OPERATION`
+    > Don't use the `CUSTOM` operation, it is not fully supported
+1. Provide the limit for max parallel instances of this pipeline
```

Comment:
`max parallel instances`?
I'd recommend mentioning the exact name of the field in the form on which the field is displayed.

<img width="846" alt="image" src="https://github.com/devopness/devopness/assets/1943318/8ef06262-919b-4ed7-8ecd-2256a735cc48">